code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -54,8 +54,11 @@
 
 /obj/structure/transit_tube_pod/ex_act(severity, target)
 	. = ..()
-	if(!QDELETED(src))
-		empty_pod()
+	if(QDELETED(src))
+		return FALSE
```

Comment:
how is this resisting the explosion? it was deleted due to the explosion, why wouldnt it return true?

Reply:
> how is this resisting the explosion? this checks if it was deleted due to the explosion from parent, why wouldnt it return true?